Books/client/src/styles/_variables.scss

```diff
@@ -1,18 +1,19 @@
 /*colors */
 $clr-aqua-deep: 177 100% 14%;
 $clr-white: 60 100% 100%;
-$clr-jungle-mist: 177 100% 14%;
+$clr-jungle-mist: 163 29% 75%;
 $clr-saffron-mango: 36 93% 68%;
+$clr-chardonnay: 36 93% 77%;
 $clr-black-pearl: 178 100% 6%;
 $clr-sherwood-green: 177 100% 10%;
 
 /* font sizes */
-$fs-400: 1.125rem;
-$fs-500: 1.25rem;
-$fs-600: 2rem;
-$fs-700: 2.75rem;
-$fs-800: 3rem;
-$fs-900: 3.5rem;
+$fs-400: 18px;
+$fs-500: 20px;
+$fs-600: 24px;
+$fs-700: 30px;
+$fs-800: 36px;
+$fs-900: 48px;
```

Comment:
Using fixed pixels instead of rem is actually bad practice for responsive, if you wanna more information here:
https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/

Reply:
I normally use rem fo font values but forgot to change this back after copying it from Figma, thanks for the reminder, I've updated my code accordingly.